src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/DataAccess.scala

```diff
@@ -23,6 +23,13 @@ trait DataAccess
   
   import driver.api._
 
+  lazy val createTempTables = Seq(
+    """create temporary table EXPREVAL_TEMP (id bigint(20) unsigned NOT NULL, name VARCHAR(254) NOT NULL, transaction_id CHAR(36) NOT NULL)""",
+    """create temporary table ENTITY_ATTRIBUTE_TEMP (id bigint(20) unsigned NOT NULL AUTO_INCREMENT primary key, name text NOT NULL, value_string text, value_number double DEFAULT NULL, value_boolean bit(1) DEFAULT NULL, value_entity_ref bigint(20) unsigned DEFAULT NULL, list_index int(11) DEFAULT NULL, list_length int(11) DEFAULT NULL, owner_id bigint(20) unsigned NOT NULL, transaction_id CHAR(36) NOT NULL)""",
+    """create temporary table SUBMISSION_ATTRIBUTE_TEMP (id bigint(20) unsigned NOT NULL AUTO_INCREMENT primary key, name text NOT NULL, value_string text, value_number double DEFAULT NULL, value_boolean bit(1) DEFAULT NULL, value_entity_ref bigint(20) unsigned DEFAULT NULL, list_index int(11) DEFAULT NULL, list_length int(11) DEFAULT NULL, owner_id bigint(20) unsigned NOT NULL, transaction_id CHAR(36) NOT NULL)""",
+    """create temporary table WORKSPACE_ATTRIBUTE_TEMP (id bigint(20) unsigned NOT NULL AUTO_INCREMENT primary key, name text NOT NULL, value_string text, value_number double DEFAULT NULL, value_boolean bit(1) DEFAULT NULL, value_entity_ref bigint(20) unsigned DEFAULT NULL, list_index int(11) DEFAULT NULL, list_length int(11) DEFAULT NULL, owner_id binary(16) NOT NULL, transaction_id CHAR(36) NOT NULL)"""
+  ).mkString("; ")
+
```

Comment:
it is just this usage. I tried to get around it using a stored procedure but had a chicken/egg problem: liquibase tries to create the procedure which needs a connection which tries to run the procedure which hasn't been created yet. The current approach was easier than figuring out how to break that dependency, but I can look into creating a special connection for liquibase.